edge-modules/api-proxy-module/templates/nginx_default_config.conf

```diff
@@ -26,6 +26,8 @@ http {
 
         chunked_transfer_encoding on;
 
+        	
+        ssl_protocols TLSv1.2;
```

Comment:
We need to make this configurable to allow setting to TLSv1.2, but keep the default.

Reply:
Otherwise it will be a breaking change.

Reply:
Also, does nginx allow setting the ciphers as well, so that the customer can limit the ones used? 
Something like this - https://github.com/Azure/iotedge/tree/master/edge-modules/edgehub-proxy

Reply:
@varunpuranik So, default to 1.2, or default to the underlying library's default?

Reply:
Yes I can put cyphers too